symfony/ux-meta/1.0/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+    "bundles": {
+        "Symfony\\WebpackEncoreBundle\\WebpackEncoreBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "assets/": "assets/",
+        "config/": "%CONFIG_DIR%/",
+        "package.json": "package.json",
+        "webpack.config.js": "webpack.config.js"
+    },
+    "aliases": ["ux", "encore", "webpack", "webpack-encore"],
```

Comment:
The webpack and encore aliases should not be added on this ux-meta package (and `ux` should be removed from the WebpackEncoreBundle recipe)